src/ts/components/seekbarlabel.ts

```diff
@@ -130,6 +130,7 @@ export class SeekBarLabel extends Container<SeekBarLabelConfig> {
    */
   setText(text: string) {
     this.timeLabel.setText(text);
+    this.timeLabel.getDomElement().attr('aria-label', `Thumbnail time: ${text}`);
```

Comment:
I'm not sure if these texts are defined somewhere.
```suggestion
    this.timeLabel.getDomElement().attr('aria-label', `Thumbnail at time: ${text}`);
```